.circleci/config.yml

```diff
@@ -73,4 +105,11 @@ workflows:
       - build:
           matrix:
             parameters:
-              install-type: ["fresh_install", "upgrade"]
+              build-type: ["rpm"]
+
+      - test:
+          matrix:
+            parameters:
+              install-type: ["rpm"]
```

Comment:
Why is the semantic meaning of install-type changed? It use to distingush between  fresh install and an upgrade, but now it looks like it is set to something completely different.

Reply:
Attempt to resolve in consecutive commits starting at https://github.com/ubccr/supremm/pull/259/commits/701e98cd093480c2693c8031cfd85cadc938d92c and ending at https://github.com/ubccr/supremm/pull/259/commits/2f4a8ea1af540891c311dc82746c16604b8e0718. Testing step now only does tests for a given installation parameter (rpm, wheel, or source).